setup.py

```diff
@@ -30,19 +30,23 @@
 		'Framework :: Django :: 2.0',
 		'Framework :: Django :: 2.1',
 		'Framework :: Django :: 2.2',
+		'Framework :: Django :: 3.0',
 		'Intended Audience :: Developers',
 		'License :: OSI Approved :: MIT License',
 		'Operating System :: OS Independent',
 		'Programming Language :: Python',
 		'Programming Language :: Python :: 3',
 		'Programming Language :: Python :: 3.4',
 		'Programming Language :: Python :: 3.5',
+		'Programming Language :: Python :: 3.6',
+		'Programming Language :: Python :: 3.7',
+		'Programming Language :: Python :: 3.8',
 		'Topic :: Database',
 		'Topic :: Internet :: WWW/HTTP',
 		'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
 	],
 	install_requires=[
-		'Django >= 1.10, < 3.0',
+		'Django >= 1.10, <= 3.0',
```

Comment:
Perhaps simply keep it `Django >= 1.10` or alternatively `Django >= 1.10, < 4.0` as a more defensive approach following semver.